assets/docs/steinding/models/yolov5/1.md

```diff
@@ -0,0 +1,51 @@
+# Module steinding/yolov5/1
+
+Object detection model pretrained on YoloV5. 
+<!-- asset-path: https://drive.google.com/drive/folders/1UGiX2CpMVpgRZY1FsdfjOlV9LU9An2HG?usp=sharing -->
```

Comment:
Could you host it as a tar.gz archive somewhere where the URL ends in `tar.gz`? Previous publishers e.g. used GCS

Reply:
Done, now changed to some GCP storage url